core/spec.md

```diff
@@ -436,6 +445,114 @@ attributes. However they MUST adhere to the following rules:
+uniquely identifies an entity within an xRegistry. Its purpose is to
+reference the entity and express a relation to it, even if it is not stored in
+the local registry. The `xid` can also be used in other places where it
+is helpful to use a URI, e.g. events or knowledge graphs.
+Each entity has exactly one `xid`. Therefore, comparing two `xid`
+values is possible to determine, if they identify the same or different
+entities. To achieve this, an xRegistry-specific URI scheme `xreg` is defined:
+
+`xid` = `"xreg://" authority "/" entity-path "/" entity-id`
+
+`entity-path` = `group-path` / `resource-path` / `version-path`
+
+`group-path`= `GROUPs "/" gID`
+
+`resource-path`= `group-path "/" RESOURCEs "/" rID`
+
+`version-path`= `resource-path "/" versions "/" vID`
+
+Where:
+
+- `GROUPs` is a Group type name (plural). e.g. `endpoints`.
+- `GROUP`, not shown, is the singular name of a Group type.
+- `gID` is the `id` of a single Group.
+- `RESOURCEs` is a Resource type name (plural). e.g. `definitions`.
+- `RESOURCE`, not shown, is the singular name of a Resource type.
+- `rID` is the `id` of a single Resource.
+- `vID` is the `id` of a single Version of a Resource.
+
+##### `authority`
+
+The `authority` is a string that uniquely identifies the entity's authority.
+It could be regarded as a namespace and SHOULD be based on a unique name
+like an entry in the domain name system (DNS) that is under the control of
+the entity's publisher.
+
+##### `entity-path`
+
+As xRegistry entities are organized in a hierarchy, the `entity-path`
+outlines the entity's position in the hierarchy. As the ABNF above shows,
+there can be paths for groups, resources, and versions.
+
+##### `entity-id`
+
+##### De-referencing a relative URI reference
+
+A relative URI reference can only be de-referenced if the base URI is known
+from the context.
+
+If the relative URI reference consists of a relative path segment, i.e. it
+does not begin with a slash, then the base URI is the URI of the surrounding
+entity.
+
+__Example:__
+
+```
+xid of surrounding entity: xreg://example.com/messagegroups/group2
+
+Reference: messages/message1
+
+De-referenced URI: xreg://example.com/messagegroups/group2/messages/message1
+```
+
+If the relative URI reference consists of an absolute path segment, i.e. it
+begins with a slash, then the base URI is the authority of the surrounding
+group.
+
+__Example:__
+
+```
+xid of surrounding group: xreg://example.com/messagegroups/group2
+
+Reference: /messagegroups/group1/messages/message1
+
+De-referenced URI: xreg://example.com/messagegroups/group1/messages/message1
+
+
+```
+
+##### Binding an `xid`
+
+As an `xid` is an abstract identifier of an entity, it is necessary to
+transform it into URLs or URL references to access the entity.
+
+For the HTTP API, the information from an `xid` can be used to construct a
+URL like this:
+
+`<Registry-base-URL>/GROUPs/{authority}${gID}/RESOURCEs/{rID}/versions/{vID}`
```

Comment:
Why is "{authority}" after GROUPs?

Reply:
I kind of expected something like: `<baseURL>/AUTHORITY$GROUPs/gID/RESOURCEs/rID/versions/vID`

Reply:
This would essentially introduce a namespace concept at the level of the registry, while my proposal choses the group level. This is of course a valid question to discuss. If we replicate metadata, do we expect to separate it on registry level, or would a user prefer to browse all resources (replicated and local) as part of a single registry?

Reply:
An idea from the 10/8 call:
```
<baseURL>/GROUPs/gID@AUTHORITY/RESOURCEs/rID/versions/vID
```

Reply:
I adjusted this PR accordingly, but we had further discussions about authorities for models **and** data. I captured some notes about that discussion here: #181